Resources/Storage/OccupancyPlugins/WLCGAccountingJson.py

```diff
@@ -0,0 +1,121 @@
+"""
+  Defines the plugin to take storage space information given by WLCG Accounting Json
+  https://twiki.cern.ch/twiki/bin/view/LCG/AccountingTaskForce#Storage_Space_Accounting
+  https://twiki.cern.ch/twiki/pub/LCG/AccountingTaskForce/storage_service_v4.txt
+  https://docs.google.com/document/d/1yzCvKpxsbcQC5K9MyvXc-vBF1HGPBk4vhjw3MEXoXf8
+"""
+import json
+import gfal2  # pylint: disable=import-error
+import os
+import tempfile
+import shutil
+
+from DIRAC import gLogger, gConfig
+from DIRAC import S_OK, S_ERROR
+
+
+class WLCGAccountingJson(object):
+  """ .. class:: WLCGAccountingJson
+
+  Occupancy plugin to return the space information given by WLCG Accouting Json
+  """
+  def __init__(self, se):
+    self.se = se
+    self.log = se.log.getSubLogger('WLCGAccountingJson')
+    self.name = self.se.name
+
+  def getOccupancy(self, **kwargs):
+    """ Returns the space information given by LCG Accouting Json
+
+        :returns: S_OK with dict (keys: SpaceReservation, Total, Free)
+    """
+    occupancyLFN = kwargs['occupancyLFN']
+
+    if not occupancyLFN:
+      return S_ERROR("Failed to get occupancyLFN")
+
+    tmpDirName = tempfile.mkdtemp()
+    filePath = os.path.join(tmpDirName, os.path.basename(occupancyLFN))
+
+    for storage in self.se.storages:
+      try:
+        ctx = gfal2.creat_context()
+        params = ctx.transfer_parameters()
+        params.overwrite = True
+        res = storage.updateURL(occupancyLFN)
+        if not res['OK']:
+          return res
```

Comment:
Maybe here instead of failing immediately and returning, we could just do `continue`, so we try with the next storage plugin